integration/presto/src/main/java/org/apache/carbondata/presto/CarbonVectorBatch.java

```diff
@@ -102,6 +89,12 @@ public static CarbonColumnVectorImpl createDirectStreamReader(int batchSize, Dat
       } else {
         return null;
       }
+    } else if (DataTypes.isArrayType(field.getDataType())) {
+      if (field.getChildren().size() > 1) {
```

Comment:
remove this assert, array can never have more than one child

Reply:
done